kernel/src/process.rs

```diff
@@ -39,6 +39,47 @@ extern "C" {
 
 pub static mut PROCS: &'static mut [Option<Process<'static>>] = &mut [];
 
+/// Helper function to load processes from flash into an array of active
+/// processes. This is the default template for loading processes, but a board
+/// is able to create its own `load_processes()` function and use that instead.
+pub unsafe fn load_processes(num_procs: usize,
+    app_memory: &mut [u8],
+    procs: &'static mut [Option<Process<'static>>],
+    fault_response: FaultResponse) -> &'static mut [Option<Process<'static>>] {
```

Comment:
why return anything? isn't just the same as `procs`? I believe returning something here technically results in an alias, which is bad...